sli/tokenstack.h

```diff
@@ -98,7 +98,8 @@ class TokenStack : private TokenArrayObj
     pop_back();
   }
 
-  void pop_move( Token& e ) // new one 5.5.97
+  void
+  pop_move( Token& e ) // new one 5.5.97
```

Comment:
What's this number supposed to mean?
```suggestion
  pop_move( Token& e )
```

Reply:
> What's this number supposed to mean?

... that the `pop_move(Token&)` method was introduced to the SLI interpreter on the fifth day of the month of May in the 1997th year of our current calendar system. Given that we brutally severed—for good reasons—our version control history when moving to Github, comments like this one are important reminders of the long history of NEST in a previous millenium and I would be reluctant to remove it ;).